vslib/src/VirtualSwitchSaiInterface.cpp

```diff
@@ -817,6 +819,25 @@ sai_status_t VirtualSwitchSaiInterface::objectTypeGetAvailability(
         *count = 3;
         return SAI_STATUS_SUCCESS;
     }
+    // MPLS Inseg and MPLS NH CRM use sai_object_type_get_availability() API.
+    else if ((objectType == SAI_OBJECT_TYPE_INSEG_ENTRY) ||
+             ((objectType == SAI_OBJECT_TYPE_NEXT_HOP) && attrList &&
+              (attrList[0].id == SAI_NEXT_HOP_ATTR_TYPE) &&
```

Comment:
you don't check attrCount whether is zero or not, and what if attrCount is 2 and Ot == SAI_OBJECT_TYPE_NEXT_HOP, because user may request count for more than one attribute at a time

Reply:
@kcudnik Added check for (attrCount == 1).